src/isomorphic/children/ReactChildren.js

```diff
@@ -11,39 +11,251 @@
 
 'use strict';
 
-var PooledClass = require('PooledClass');
 var ReactElement = require('ReactElement');
 
 var emptyFunction = require('fbjs/lib/emptyFunction');
-var traverseAllChildren = require('traverseAllChildren');
+var invariant = require('fbjs/lib/invariant');
 
-var twoArgumentPooler = PooledClass.twoArgumentPooler;
-var fourArgumentPooler = PooledClass.fourArgumentPooler;
+if (__DEV__) {
+  var warning = require('fbjs/lib/warning');
+  var {getStackAddendum} = require('ReactDebugCurrentFrame');
+}
+
+var ITERATOR_SYMBOL = typeof Symbol === 'function' && Symbol.iterator;
+var FAUX_ITERATOR_SYMBOL = '@@iterator'; // Before Symbol spec.
+// The Symbol used to tag the ReactElement type. If there is no native Symbol
+// nor polyfill, then a plain number is used for performance.
+var REACT_ELEMENT_TYPE =
+  (typeof Symbol === 'function' && Symbol.for && Symbol.for('react.element')) ||
+  0xeac7;
+
+var SEPARATOR = '.';
+var SUBSEPARATOR = ':';
+
+/**
+ * Escape and wrap key so it is safe to use as a reactid
+ *
+ * @param {string} key to be escaped.
+ * @return {string} the escaped key.
+ */
+function escape(key: string): string {
```

Comment:
Hey,
Why you added types in flow style if this file have't flow comment?

Reply:
Probably accidentally or by copy paste. Want to send a PR to Flow-ify it?

Reply:
yep! I can delete useless flow typechecking or add comment flow. If I add flow comment maybe I can try to update types for all file? If you want this.

Reply:
Sure.